<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgBundleDialog</class>
 <widget class="QDialog" name="HgBundleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercurial Bundle</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Revision</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="multipleButton">
        <property name="toolTip">
         <string>Select to specify multiple revisions</string>
        </property>
        <property name="text">
         <string>Revisions:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="multipleEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter revisions by number, id, range or revset expression one per line</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="tagButton">
        <property name="toolTip">
         <string>Select to specify a revision by a tag</string>
        </property>
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="tagCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a tag name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="branchButton">
        <property name="toolTip">
         <string>Select to specify a revision by a branch</string>
        </property>
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="branchCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a branch name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="bookmarkButton">
        <property name="toolTip">
         <string>Select to specify a revision by a bookmark</string>
        </property>
        <property name="text">
         <string>Bookmark:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="bookmarkCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a bookmark name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QRadioButton" name="noneButton">
        <property name="toolTip">
         <string>Select to not specify a specific revision</string>
        </property>
        <property name="text">
         <string>No revision selected</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Base Revisions:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="baseRevisionsEdit">
       <property name="toolTip">
        <string>Enter changesets by number, id, range or revset expression one per line</string>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Compression:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="compressionCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the compression type (empty for default)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="allCheckBox">
     <property name="toolTip">
      <string>Select to bundle all changesets</string>
     </property>
     <property name="text">
      <string>Bundle all changesets</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>multipleButton</tabstop>
  <tabstop>multipleEdit</tabstop>
  <tabstop>tagButton</tabstop>
  <tabstop>tagCombo</tabstop>
  <tabstop>branchButton</tabstop>
  <tabstop>branchCombo</tabstop>
  <tabstop>bookmarkButton</tabstop>
  <tabstop>bookmarkCombo</tabstop>
  <tabstop>noneButton</tabstop>
  <tabstop>baseRevisionsEdit</tabstop>
  <tabstop>compressionCombo</tabstop>
  <tabstop>allCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgBundleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgBundleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tagButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tagCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branchButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>branchCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarkButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>bookmarkCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>multipleEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
